src/Uno.Toolkit.UI/Behaviors/FlipViewExtensions.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Input;
+using Microsoft.Extensions.Logging;
+using Uno.Extensions;
+using Uno.Logging;
+using Windows.System;
+using Windows.UI.ViewManagement;
+
+#if IS_WINUI
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Controls.Primitives;
+using Microsoft.UI.Xaml.Controls;
+using Microsoft.UI.Xaml.Input;
+#else
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls.Primitives;
+using Windows.UI.Xaml.Input;
+#endif
+
+namespace Uno.Toolkit.UI;
+
+public static class FlipViewExtensions
+{
+	public static DependencyProperty NextProperty { get; } =
```

Comment:
consider using the same boilerplate code for DPs:

```cs
#region DependencyProperty: Next

public static DependencyProperty NextProperty { get; } = DependencyProperty.RegisterAttached(
	"Next",
	typeof(FlipView),
	typeof(FlipViewExtensions),
	new PropertyMetadata(default(FlipView), OnNextChanged));

public static FlipView GetNext(Button obj) => (FlipView)obj.GetValue(NextProperty);
public static void SetNext(Button obj, FlipView value) => obj.SetValue(NextProperty, value);

#endregion
#region DependencyProperty: Previous

public static DependencyProperty PreviousProperty { get; } = DependencyProperty.RegisterAttached(
	"Previous",
	typeof(FlipView),
	typeof(FlipViewExtensions),
	new PropertyMetadata(default(FlipView), OnPreviousChanged));

public static FlipView GetPrevious(Button obj) => (FlipView)obj.GetValue(PreviousProperty);
public static void SetPrevious(Button obj, FlipView value) => obj.SetValue(PreviousProperty, value);

#endregion
```

```cs
private static void OnNextChanged(DependencyObject sender, DependencyPropertyChangedEventArgs e) => throw new NotImplementedException();

private static void OnPreviousChanged(DependencyObject sender, DependencyPropertyChangedEventArgs e) => throw new NotImplementedException();
```

and, prefer to not put the on-change handler in the `#region`
the idea is to make the file much easier to scan when folded to lv3:
![image](https://user-images.githubusercontent.com/2359550/196461439-6307bb4f-91e9-4275-9b9b-52e1e78399e8.png)
^ DrawerControl.Properties.cs

Reply:
the code generator can be found here: https://github.com/Xiaoy312/bag-o-scripts/blob/20221018-code-dump/DependencyProperty%20codegen.linq